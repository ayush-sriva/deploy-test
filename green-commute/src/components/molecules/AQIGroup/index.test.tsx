import renderer from 'react-test-renderer';
import AQIIndexLocation from '.';

const data = [
  {
    id: 'AQIL01',
    index: '894',
    location: 'Hyderabad',
  },
  {
    id: 'AQIL02',
    index: '900',
    location: 'Mumbai',
  },
];

describe('AQIIndexLocation', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AQIIndexLocation children='description' data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
