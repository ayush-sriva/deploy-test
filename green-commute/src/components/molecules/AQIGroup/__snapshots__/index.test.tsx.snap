// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AQIIndexLocation renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container makeStyles-container-1 css-11lq3yg-MuiGrid-root"
>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-item-2 css-13i4rnv-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-container makeStyles-container-4 css-11lq3yg-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item makeStyles-aqi-5 css-13i4rnv-MuiGrid-root"
      >
        <span
          className="MuiTypography-root MuiTypography-body3 makeStyles-innerText-6 css-37jdci-MuiTypography-root"
        >
          894
        </span>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item makeStyles-item-8 css-13i4rnv-MuiGrid-root"
      >
        <h3
          className="MuiTypography-root MuiTypography-h3 makeStyles-text-7 css-gepadz-MuiTypography-root"
        >
          Hyderabad
        </h3>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-item-2 css-13i4rnv-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-container makeStyles-container-4 css-11lq3yg-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item makeStyles-aqi-5 css-13i4rnv-MuiGrid-root"
      >
        <span
          className="MuiTypography-root MuiTypography-body3 makeStyles-innerText-6 css-37jdci-MuiTypography-root"
        >
          900
        </span>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item makeStyles-item-8 css-13i4rnv-MuiGrid-root"
      >
        <h3
          className="MuiTypography-root MuiTypography-h3 makeStyles-text-7 css-gepadz-MuiTypography-root"
        >
          Mumbai
        </h3>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-text-3 css-13i4rnv-MuiGrid-root"
  >
    <h2
      className="MuiTypography-root MuiTypography-h2 css-1sra7t5-MuiTypography-root"
    >
      description
    </h2>
  </div>
</div>
`;
